bower.json

```diff
@@ -37,9 +37,9 @@
     "iron-test-helpers": "^2.0.0",
     "webcomponentsjs": "^1.0.0",
     "web-component-tester": "^6.1.5",
-    "vaadin-text-field": "vaadin/vaadin-text-field#^2.1.0",
-    "vaadin-ordered-layout": "vaadin/vaadin-ordered-layout#^1.1.0",
-    "vaadin-demo-helpers": "vaadin/vaadin-demo-helpers#^3.0.0"
+    "vaadin-text-field": "vaadin/vaadin-text-field#^2.6.0-alpha4",
+    "vaadin-ordered-layout": "vaadin/vaadin-ordered-layout#^1.3.0-alpha1",
+    "vaadin-demo-helpers": "vaadin/vaadin-demo-helpers#^3.1.0-alpha1"
   },
   "resolutions": {
```

Comment:
do resolutions still make sense?

Reply:
Not sure, but for `next` branch that is already removed.

Reply:
Removed.